string gt_file_path
string map_file_path
---
bool success